templates/graphdb-master.yaml

```diff
@@ -116,42 +143,73 @@ spec:
           command: ['sh', '-c']
           args:
             - |
-              {{ $repo := $.Values.graphdb.masters.repository -}}
               set -eu
-              mkdir -p /opt/graphdb/home/data/repositories/{{ $repo }} ;
-              cd /opt/graphdb/home/data/repositories/{{ $repo }} ;
-
-              # If the repo configuration exists -> skip provisioning
-              [ -f config.ttl ] && echo "Repository exists, exiting..." && exit 0
+              for filename in /tmp/graphdb/*.ttl; do
```

Comment:
WDYT about moving these steps in a script mounted as a configmap ?

Reply:
As they are duplicated it's a good idea ... will create a task